### PR TITLE
Add frontend tensorflow dct

### LINKED ISSUE #6003: dct

### PR DESCRIPTION
**Pull request by MuokaPWambua**
Close #6003

### CODE REVIEW

File: ivy/functional/frontends/tensorflow/signal.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import ivy
+from ivy.functional.frontends.tensorflow.func_wrapper import to_ivy_arrays_and_back
+
+
+@to_ivy_arrays_and_back
+def dct(input, type=2, n=None, axis=-1, norm=None, name=None):
+    return ivy.dct(input, type, n, axis, norm, name)
```
Comment (Daniel4078) [inline ivy/functional/frontends/tensorflow/signal.py:7]: I do not think there is an ivy.dct function for you to use directly, please check the deep dive document https://lets-unify.ai/ivy/deep_dive/ivy_frontends.html#compositions for what to do in this case. Thank you.

Comment (MuokaPWambua) [reply ivy/functional/frontends/tensorflow/signal.py:7]: Hi Daniel, have created a new issue according to the documentation, but i was unable to add the labels as stated on the documentation, any way forward?

Comment (Daniel4078) [reply ivy/functional/frontends/tensorflow/signal.py:7]: In this case, I will add the labels for you. Can you give me the link to the issue? Thank you.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**